specification/managedservices/resource-manager/readme.md

```diff
@@ -26,18 +26,27 @@ These are the global settings for the ManagedServices API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-09
+tag: package-preview-2020-02
```

Comment:
You need to change this tag too. The SDK generation cannot find this specific tag.